### DISCUSSION TITLE
HELP PLEASE

### DISCUSSION THREAD

**Discussion by MTB-ALFIE** [fredemmott/StreamDeck-Discord#49] [Q&A] [answered]
I need help, I have entered everything correctly and on discord, I don't get a prompt I have tried everything linked on https://github.com/fredemmott/StreamDeck-Discord/blob/master/TROUBLESHOOTING.md  but still, nothing is triggering a prompt

[FIXED]

**Comment by fredemmott** [comment, marked as answer]
author marked as fixed, responding so discussion can be closes
